projectile.el

```diff
@@ -367,7 +367,7 @@ containing a root file."
   :type '(repeat string))
 
 (defcustom projectile-globally-unignored-files nil
-  "A list of files globally unignored by projectile."
+  "A list of files globally unignored by projectile. Regular expressions can be used."
```

Comment:
The second sentence should be on the next line. This applies to all the docstring changes.